docs_parser.py

```diff
@@ -62,7 +64,7 @@ def append_paragraph(lines, index, tokens):
     new_paragraph = {'type':'p','content':lines[index].strip()}
     index = index + 1
     while index < len(lines) and lines[index].strip() != '':
-        new_paragraph['content'] = new_paragraph['content'] + ' ' + lines[index].strip()
+        new_paragraph['content'] = '%s %s' * (new_paragraph['content'], lines[index].strip())
```

Comment:
Please confirm that you don't need the space between the content and the new line.